root/usr/share/nethesis/NethServer/Module/OpenVpnTunnels/Servers.php

```diff
@@ -59,6 +59,25 @@ public function initialize()
         parent::initialize();
     }
 
+    public function readDigests()
```

Comment:
Alternatives to duplicating code:
- Parent's class public method `$this->getParent()->methodName()`
- Parent's class public static method `parentClass::staticMethodName()` 
- PHP trait (for complex cases) https://github.com/search?q=org%3Anethserver+trait&type=Code http://php.net/manual/it/language.oop5.traits.php

Duplicating code can be acceptable though...